(monproc) add Centreon SQL Metrics monitoring procedure (#1462)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-monitoring-centreon-sql-metrics.md

## Contenu du Pack

### Modèles

Le Plugin Pack Centreon SQL Metrics apporte un modèle d'hôte :

* App-Monitoring-Centreon-SQL-Metrics-custom

Il apporte les modèles de service suivants :

| Alias           | Modèle de service                           | Description                                                              | Défaut  |
| :-------------- | :------------------------------------------ | :----------------------------------------------------------------------- | :------ |
| Poller-Delay    | App-Monitoring-Centreon-SQL-Poller-Delay    | Contrôle le délai dans la mise à jour des collecteurs                    |         |
| Virtual-Curve   | App-Monitoring-Centreon-SQL-Virtual-Curves  | Combine des métriques existantes et effectue des calculs supplémentaires |         |

### Métriques & statuts collectés

<Tabs groupId="sync">
<TabItem value="Poller-Delay" label="Poller-Delay">

| Metric Name                    | Unit   |
| :----------------------------- | :----- |
| centreon.poller.delay.seconds  |   s    |

</TabItem>
<TabItem value="Virtual-Curve" label="Virtual-Curve">

> Les métriques dépendent de la configuration du service. Vous pouvez consulter l'article sur [The Watch](https://thewatch.centreon.com/product-how-to-21/get-to-know-app-centreon-sql-metric-pack-and-start-building-some-virtual-curves-296)

Reviewer: ```suggestion
Les métriques dépendent de la configuration du service. Voir l'article sur [The Watch](https://thewatch.centreon.com/product-how-to-21/get-to-know-app-centreon-sql-metric-pack-and-start-building-some-virtual-curves-296).
```

3306/TCP grâce aux valeurs fournies par les options --username et --password. 

L'utilisateur doit avoir les droits de réaliser un 'SELECT' sur les tables index_data, metrics et instances de la base centreon_storage. 

Pour le service Virtual-Curve, le fichier de configuration associé doit pouvoir être lu par l'utilisateur centreon-engine.

## Installation

<Tabs groupId="sync">
<TabItem value="Online License" label="Online License">

1. Installer le Plugin Centreon sur le serveur Central :

```bash
yum install centreon-plugin-Applications-Monitoring-Centreon-SQL-Metrics
```

2. Sur l'interface Web de Centreon, installer le Plugin Pack **Centreon SQL Metrics** depuis la page **Configuration > Packs de plugins**.

</TabItem>
<TabItem value="Offline License" label="Offline License">

1. Installer le Plugin Centreon sur le serveur Central :

```bash
yum install centreon-plugin-Applications-Monitoring-Centreon-SQL-Metrics
```

2. Sur le serveur Central, installer le RPM du Pack **Centreon SQL Metrics** :

```bash
yum install centreon-pack-applications-monitoring-centreon-poller
```

> 3. Sur l'interface Web de Centreon, installer le Plugin Pack **Centreon SQL Metrics** depuis la page **Configuration > Packs de plugins**.

Reviewer: ```suggestion
3. Sur l'interface web de Centreon, installer le Plugin Pack **Centreon SQL Metrics** depuis la page **Configuration > Packs de plugins**.
```

</TabItem>
</Tabs>

## Configuration

### Hôte

> * Ajoutez un Hôte à Centreon depuis la page **Configuration > Hôtes**.

Reviewer: ```suggestion
* Ajoutez un hôte à Centreon depuis la page **Configuration > Hôtes**.
```

> * Complétez les champs **Nom**, **Alias** & **IP Address/DNS** correspondant à votre serveur de base de données Centreon. 

Reviewer: ```suggestion
* Complétez les champs **Nom**, **Alias** et **IP Address/DNS** correspondant à votre serveur de base de données Centreon. 
```

* Appliquez le modèle d'hôte **App-Monitoring-Centreon-SQL-Metrics-custom**.

* Une fois le modèle appliqué, les macros ci-dessous indiquées comme requises (*Obligatoire*) doivent être renseignées.

| Obligatoire | Nom                      | Description                                      |
| :---------- | :----------------------- | :----------------------------------------------- |
|     x       | CENTREONDATABASEUSER     | Nom d'utilisateur pour la base centreon_storage  |
|     x       | CENTREONDATABASEPASSWORD | Mot de passe pour la base centreon_storage       |
|             | EXTRAOPTIONS             | Options supplémentaires pour les contrôles       |

> ## Comment puis-je tester le Plugin et que signifient les options des commandes ? 

Reviewer: ```suggestion
## Comment puis-je tester le plugin et que signifient les options des commandes ? 
```
